vue-components/src/components/ScatterPlot.vue

```diff
@@ -84,20 +84,10 @@ onMounted(() => {
       return `rgba(25,118,210,${alpha})`
     },
     onHover(index: number | null) {
-      if (index) {
-        if (index >= props.points.length) {
-          const selectedPointsIdx = index - props.points.length
-          index = props.selectedPoints[selectedPointsIdx]
-        }
-      }
       emit('hover', index)
     },
     onSelect(indices) {
-      onPanModeClick()
```

Comment:
This will be good to keep

Reply:
it will be also nice to do a `onPanModeClick` also after selection of multiple points

Reply:
I think we should not move back to camera move mode after selection.  That surprised me when using app first time.

Reply:
The idea behind that was that after you select you move to the default mode that was camera mode, in that manner you can pan|rotate to see how the transformations embedding PCA or similar differs from the original. So select mode is only active until there is a selection. It felt as an sort of standard selection box tool works. Think when selecting multiple files in a file manager for example.

Reply:
Thanks for the reasoning.  I put the behavior back.  Still not convinced but no biggie =)

Reply:
I backed out this change.  Anything else before merging?